server/src/bcaap-application/bcaap-application.entity.ts

```diff
@@ -0,0 +1,54 @@
+import { User } from '@/user/user.entity';
+import { Entity, PrimaryGeneratedColumn, Column, JoinColumn, OneToOne } from 'typeorm';
+import { CustomBaseEntity } from '../common/custom-base.entity';
+import {
+  ContactInfo,
+  GeneralProjectInfo,
+  FacilityInfo,
+  FundingEligibility,
+  ClimateConsiderationsAirside,
+  ClimateConsiderationsEnvironmental,
+  ProjectBenefits,
+  ProjectFunding,
+  SupportAndAuthorization,
+} from './bcaap-application.interfaces';
+
+@Entity()
+export class BCAAPApplication extends CustomBaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'int', nullable: true })
+  priority: number;
```

Comment:
What's priority?

Reply:
<img width="794" alt="image" src="https://user-images.githubusercontent.com/102998728/196234897-454e405b-5668-49c6-babe-8a4cca843ce6.png">

It's supposed to be this, but since it's a part of Contact Info, I'll move it there.